lib/fs/fakefs.go

```diff
@@ -299,7 +320,12 @@ func (fs *fakefs) Mkdir(name string, perm FileMode) error {
 		return os.ErrExist
 	}
 
-	entry.children[base] = &fakeEntry{
+	key := base
+	if fs.insens {
+		key = UnicodeLowercase(key)
+	}
```

Comment:
This breaks "already exists" for case insensitive cases. Suggest you add a test.

Reply:
You're right. Wrote a test, will work on it.

Reply:
Solved.